lib/seccomp/unittest/Makefile

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+include $(CPP_SRC_HOME)/mk/defines.mk
+
+TARGET=ml_test$(EXE_EXT)
+
+USE_BOOST=1
+USE_BOOST_REGEX_LIBS=1
```

Comment:
You shouldn't need this as `lib/core` has it and only `lib/core` is using the Boost regex lib directly.  (If you do need it then it's a sign something isn't working as expected.)